### PR TITLE
Update DataflowRunner to suppress duplicate system messages

### PR DESCRIPTION
**Pull request by charlesccychen**
This change fixes duplicate logging in the DataflowRunner at HEAD.

### CODE REVIEW

**Comment by charlesccychen** [comment]
R: @aaltay

File: sdks/python/apache_beam/runners/dataflow/dataflow_runner.py [hunk 1]
```diff
@@ -159,15 +159,18 @@ def rank_error(msg):
...
-          if last_message_hash is not None and m_hash == last_message_hash:
-            # Skip the first message if it is the last message we got in the
-            # previous round. This can happen because we use the
-            # last_message_time as a parameter of the query for new messages.
+          if m.time > last_message_time:
+            last_message_time = m.time
+            current_seen_messages = set()
+
+          if message in current_seen_messages:
+            # Skip the first message if it has already been seen at the current
```
Comment (aaltay) [inline sdks/python/apache_beam/runners/dataflow/dataflow_runner.py]: drop `first`?

Comment (charlesccychen) [reply sdks/python/apache_beam/runners/dataflow/dataflow_runner.py]: Done.

**Comment by charlesccychen** [comment]
Thanks, PTAL.

**Comment by aaltay** [comment]
retest this please

**Comment by charlesccychen** [comment]
Rebased to head, PTAL.

**Comment by charlesccychen** [comment]
ping @aaltay

**Pull request merged**